code/game/objects/items/weapons/storage/briefcase.dm

```diff
@@ -20,10 +20,78 @@
 	force = 10
 
 /obj/item/storage/briefcase/sniperbundle/New()
-	..() 
+	..()
 	new /obj/item/gun/projectile/automatic/sniper_rifle/syndicate(src)
 	new /obj/item/clothing/accessory/red(src)
 	new /obj/item/clothing/under/syndicate/sniper(src)
 	new /obj/item/ammo_box/magazine/sniper_rounds/soporific(src)
 	new /obj/item/ammo_box/magazine/sniper_rounds/soporific(src)
-	new /obj/item/suppressor/specialoffer(src)
\ No newline at end of file
+	new /obj/item/suppressor/specialoffer(src)
+
+/obj/item/storage/briefcase/false_bottomed
+	name = "briefcase"
+	desc = "It's made of AUTHENTIC faux-leather and has a price-tag still attached. This one feels a bit heavier than normal for how much fits in it."

```

Comment:
I'd rather the description be identical to the regular briefcase so people can't meta it PURELY by looking at it.